content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -3,17 +3,184 @@ title: 'MongoDB'
+| `tlsCertificateKeyFilePassword` |         | Password for the `tlsCertificateKeyFile`. New in MongoDB version 4.2.                                                                                                                                                                                                                                                          |
+| `tlsCAFile`                     |         | Location of a local `.pem` file for a root certificate chain from the Certificate Authority. New in MongoDB version 4.2.                                                                                                                                                                                                       |
+| `tlsAllowInvalidCertificates`   |         | Bypasses the validation of the certificates used on the mongod/mongos instance. New in MongoDB version 4.2.                                                                                                                                                                                                                    |
+| `tlsAllowInvalidHostnames`      |         | Disables the hostname validation of the certificate used by the mongod/mongos instance. New in MongoDB version 4.2.                                                                                                                                                                                                            |
+| `tlsInsecure`                   |         | Disables certificate validations. New in MongoDB version 4.2.                                                                                                                                                                                                                                                                  |
+| `connectTimeoutMS`              |         | Time in milliseconds the before the connection times out.                                                                                                                                                                                                                                                                      |
+| `socketTimeoutMS`               |         | Time in milliseconds to send or receive on an open socket.                                                                                                                                                                                                                                                                     |
+| `socket_timeout`                |         | Maximum number of seconds to wait until a single query terminates.                                                                                                                                                                                                                                                             |
```

Comment:
Are these different?

Reply:
Hmm, i was sure these were two different things but i cant now find `socket_timeout` on the mongo docs...

Reply:
`socket_timeout` is how we call this param for the other connectors (MySQL, Postgres etc).

Reply:
Removed this table as we don't have any Prisma-specific params - it's all MongoDB

Reply:
Hmmmm - but those parameters are still what people need to use on the connection string they are using. We are also listing a few default MySQL and PostgreSQL parameters on our page. 

I think we should adopt that list then and show clearly which ones we support (sorry).